torch/csrc/jit/codegen/cuda/parallel_dimension_map.cpp

```diff
@@ -130,9 +130,9 @@ void ParallelDimensionMap::populateDimensionMapWithMultipleCASet(
   kir::IrBuilder ir_builder(gpu_lower->kernel());
 
   bool all_equal = true;
-  kir::Val* known_dimension =
-      gpu_lower->lowerValue((*dom_set.begin())->extent());
```

Comment:
So the main change is essentially to avoid picking up any extent that wasn't on a concretized `IterDomain`.